acceptance/bundle/includes/yml_outside_root/output.txt

```diff
@@ -0,0 +1,31 @@
+
+>>> [CLI] bundle validate
+Name: yml_outside_root
+Target: default
+Workspace:
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/yml_outside_root/default
+
+Validation OK!
+
+>>> [CLI] bundle validate -o json
+{
+  "deployment": {
+    "kind": "BUNDLE",
+    "metadata_file_path": "/Workspace/Users/[USERNAME]/.bundle/yml_outside_root/default/state/metadata.json"
+  },
+  "edit_mode": "UI_LOCKED",
+  "format": "MULTI_TASK",
+  "name": "include_outside_root",
+  "permissions": [],
+  "queue": {
+    "enabled": true
+  },
+  "tags": {},
+  "tasks": [
+    {
+      "description": "task_description",
+      "task_key": "task_key"
+    }
+  ]
+}
```

Comment:
Would also be interesting to see

```
  "bundle": {
      "bundle_root_path":
...
  "sync": {
    "paths": [
...
  "workspace": {
    "artifact_path": ...
```

Reply:
Artifact path is remote path is not really relevant here but the rest makes sense